pkg/build/webhook/github/github_test.go

```diff
@@ -234,7 +234,7 @@ func setup(t *testing.T, filename, eventType string) *testContext {
 	}
 	req, err := http.NewRequest("POST", "http://origin.com", bytes.NewReader(event))
 	req.Header.Add("Content-Type", "application/json")
-	req.Header.Add("X-Github-Event", eventType)
+	req.Header.Add("X-GitHub-Event", eventType)
```

Comment:
Probably better to revert this part, add a constant and a comment explaining this is EXACTLY what upstream sends.